src/avataaar.ts

```diff
@@ -0,0 +1,4 @@
+import publicConfig from "./publicConfig"
+
+export const avataaar = (hash: string): string =>
```

Comment:
spelling of **Avatar** is different across project files/functions

`src/avataaar.ts`

Reply:
Good catch, I updated the avataar variables to avataaar. I also renamed Avatar.tsx to AccountImage.tsx because it may contain non-avataaar images soon.

Reply:
ok, funny question but why not just correct spelling of **"avatar"** 🤔

Reply:
Haha, yes it's a bit confusing - this is using https://getavataaars.com. It's also used by flow-playground: https://github.com/onflow/flow-playground/blob/master/src/util/avataaar.ts

Should I change everything to "avatar" anyway?

Reply:
Oh I see. I do think the extra a's could lead to sp bugs and if no issue would suggest the correct spelling all around or expand to specific 
userAvatar or something like that

Reply:
Sounds good, I renamed everything to `avatar`